modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Manage/User/ChangeUserPassword.tsx

```diff
@@ -12,73 +12,68 @@
  * details.
  */
 
-import ClayButton from '@clayui/button';
 import ClayForm from '@clayui/form';
-import ClayLayout from '@clayui/layout';
+import {yupResolver} from '@hookform/resolvers/yup';
+import React from 'react';
+import {useForm} from 'react-hook-form';
 import {useNavigate} from 'react-router-dom';
 
-import Input from '../../../components/Form/Input/index';
+import Form from '../../../components/Form';
 import Container from '../../../components/Layout/Container';
 import i18n from '../../../i18n';
+import yupSchema from '../../../schema/yup';
 
 const ChangeUserPassword: React.FC = () => {
 	const navigate = useNavigate();
 
-	return (
-		<ClayLayout.Container>
-			<Container>
-				<ClayForm>
-					<ClayLayout.Row justify="start">
-						<ClayLayout.Col size={3} sm={12} xl={3}>
-							<h5 className="font-weight-normal">
-								{i18n.translate('change-password')}
-							</h5>
-						</ClayLayout.Col>
+	const {
+		formState: {errors},
+		handleSubmit,
+		register,
+	} = useForm({resolver: yupResolver(yupSchema.password)});
+
+	const _onSubmit = () => alert('successful send');
 
-						<ClayLayout.Col size={3} sm={12} xl={7}>
-							<ClayForm.Group className="form-group-sm">
-								<Input
-									label={i18n.translate('new-password')}
-									name="newpassword"
-									type="password"
-								/>
+	const inputProps = {
+		errors,
+		register,
+		required: true,
+	};
+
+	return (
+		<Container className="containerPassword d-flex flex-column">
```

Comment:
@Aline160, the pattern for naming CSS usually uses `hyphen-case` , for example

`container-password`, containerPassword is using camelCase, which is not common for CSS.

So you could use:
`container-password` or use the Component Name as css.

I suggest: 

```suggestion
		<Container className="change-user-password d-flex flex-column">
```